### PR TITLE
[CARBONDATA-214]Remove binary file and generate the binary file on fly while running the test case

### PR DESCRIPTION
**Pull request by ravipesala**
Remove binary file and generate the binary file on fly while running the test case

### CODE REVIEW

**Comment by ravipesala** [comment]
http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/286/

File: processing/src/test/java/org/apache/carbondata/test/util/StoreCreator.java [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+import org.apache.carbondata.processing.graphgenerator.GraphGenerator;
+import org.apache.carbondata.processing.graphgenerator.GraphGeneratorException;
+import org.apache.carbondata.processing.util.CarbonDataProcessorUtil;
+
+import com.google.gson.Gson;
+
+/**
+ * This class will create store file based on provided schema
+ *
+ * @author Administrator
```
Comment (chenliang613) [inline processing/src/test/java/org/apache/carbondata/test/util/StoreCreator.java]: please remove "@author Administrator"

Comment (ravipesala) [inline processing/src/test/java/org/apache/carbondata/test/util/StoreCreator.java]: Ok

**Comment by chenliang613** [comment]
LGTM

**Comment by ravipesala** [comment]
Merged to master with commitid 5894980b5661a0e656665a271de55b78323c1d67

**Pull request closed without merging**